.murdock

```diff
@@ -140,8 +145,7 @@ get_app_board_toolchain_pairs() {
 get_compile_jobs() {
     get_apps | \
         dwqc ${DWQ_ENV} -s \
-        "$0 get_app_board_toolchain_pairs \${1}" \
-        | xargs '-d\n' -n 1 echo $0 compile
```

Comment:
... makes the piping to `xargs` here this unnecessary.

Reply:
yup. the commit message of the first commit also has some explanation.